doc/install.md

```diff
@@ -42,85 +45,84 @@ SECRET_KEY = "my_key"
 
 The most important configurations to setup are:
 
-- `MASTODON_ALLOW_ANY_SITE` set to `True` so that user can login via any Mastodon API compatible sites (e.g. Mastodon/Pleroma) 
+- `MASTODON_ALLOW_ANY_SITE` set to `True` so that user can login via any Mastodon API compatible sites (e.g. Mastodon/Pleroma)
 - `REDIRECT_URIS` should be `SITE_INFO["site_url"] + "/account/login/oauth"`. If you want to run **on local**, `SITE_INFO["site_url"]` should be set to `"http://localhost/"`
 
 More details on `settings.py` in [configuration.md](configuration.md)
 
 ### 1.3 Packages and Build
-Create and use `venv` as you normally would, then install packages
-```
-python3 -m pip install -r requirements.txt
+NeoDB uses [PDM](https://pdm.fming.dev) as the package manager to install and manage dependencies.
+Please visit the [PDM installation guide](https://pdm.fming.dev/latest/#installation) to install it.
 
+Install all dependencies
+```
+pdm install
 ```
 
 Quick check
 ```
-python3 manage.py check
+pdm check
 ```
 
 Initialize database
 ```
-python3 manage.py migrate
+pdm db
```

Comment:
I think it may be better to use `pdm initdb`.

Reply:
+1

Reply:
Or `pdm dbmigrate`, like `dbshell`